tests/AutoReview/FixerFactoryTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\AutoReview;
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @coversNothing
+ * @group auto-review
+ */
+final class FixerFactoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
my reasoning:
`FixerTest` checks fixer atomically one by one.
`FixerFactoryTest` checks relations between fixers (which is used only inside FixerFactory, that's why it was put here in the first place)